### PR TITLE
updated gravel beta profile

### PR DESCRIPTION
**Pull request by dhanek**
now with penalties for inclines over 15% and downhills under -10%, hiking and mtb scale penalties and a slight benefit for routes from cycling networks

also signal penalties and barrier penalties for nodes

### CODE REVIEW

**Comment by lefou** [comment]
`-10%` for downhill looks a bit small to me. Why not select `-15%` in alignment with uphill?

**Comment by dhanek** [comment]
> `-10%` for downhill looks a bit small to me. Why not select `-15%` in alignment with uphill?

After spending 3 weeks in a region with really steep uphills and downhills I found that it's okay to push uphill to about 15% incline with good gearing (or push the bike if you get stuck), but when descending on a narrow or technical trail it's neither safe nor fun if it's steeper than -10% on a gravel bike, because of the high centre of gravity compared to MTBs. An even better approach would be to add this uphill / downhill penalty depending on road type, maybe that's possible in a future version
Also it's a gradual penalty, so it starts at 10% and the steeper it gets the higher the penalty (I couldn't find a link to the actual description of how the calculation works). So even with this penalty I had some really steep descends that were barely ridable.

**Comment by lefou** [comment]
Makes sense, if this does not result in avoiding those tracks at any costs. Thanks!

File: gravel_beta.brf [hunk 1]
```diff
@@ -37,7 +42,9 @@ assign any_cycleway or cycleway=track|lane|shared_lane|shared or cycleway_right
 assign any_cycleroute or route_bicycle_icn=yes or route_bicycle_ncn=yes or route_bicycle_rcn=yes route_bicycle_lcn=yes
 assign is_paved highway=primary|primary_link|secondary|secondary_link|tertiary|tertiary_link|road|residential|unclassified|living_street|service|cycleway
 
+assign cycleroute_benefit 0.75
```
Comment (stefankeidel) [inline gravel_beta.brf:45]: Make this configurable as parameter? For example, in the zossebart profiles:

`assign mtb_hard_factor          = 2               # %mtb_hard_factor% | Difficulty of the route (0:xc/am...2:allmountain...4:enduro) | number`

**Review by stefankeidel — APPROVED** [review]

**Pull request merged**